src/Symfony/Component/DependencyInjection/Compiler/ResolveInstanceofConditionalsPass.php

```diff
@@ -63,7 +60,10 @@ private function processDefinition(ContainerBuilder $container, $id, Definition
 
         $definition->setInstanceofConditionals(array());
         $parent = $shared = null;
-        $instanceofTags = array();
+        $instanceof = array(
```

Comment:
another variable instead of an array would be better I think

Reply:
Fixed in `b61d981`